sap_hana/metadata.csv

```diff
@@ -0,0 +1,62 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+sap_hana.backup.latest,gauge,,second,,The time elapsed since the latest database backup was completed.,-1,sap_hana,
+sap_hana.connection.idle,gauge,,connection,,The current number of idle connections.,0,sap_hana,
+sap_hana.connection.open,gauge,,connection,,The current number of connections.,0,sap_hana,
+sap_hana.connection.running,gauge,,connection,,The current number of running connections.,0,sap_hana,
+sap_hana.cpu.service.utilized,gauge,,percent,,The CPU utilization of services as a percentage.,0,sap_hana,
+sap_hana.disk.free,gauge,,byte,,The total free size of the disk in bytes.,1,sap_hana,
+sap_hana.disk.size,gauge,,byte,,The total size of the disk in bytes.,0,sap_hana,
+sap_hana.disk.used,gauge,,byte,,The total used size of the disk in bytes.,-1,sap_hana,
+sap_hana.disk.utilized,gauge,,percent,,The utilization of the disk as a percentage.,-1,sap_hana,
+sap_hana.file.service.open,gauge,,thread,,The number of open files.,-1,sap_hana,
```

Comment:
```suggestion
sap_hana.file.service.open,gauge,,file,,The number of open files.,-1,sap_hana,
```

Should the unit be `file` instead of `thread` ?

Reply:
Good catch